package.json

```diff
@@ -18,16 +18,16 @@
     "eslint": "lerna exec --stream --no-bail -- eslint src/**/*.ts",
     "start": "run-p watch \"tinymce-grunt start\"",
     "dev": "npm-run-all oxide-icons-build oxide-build -p tsc \"tinymce-grunt dev\"",
-    "ci": "npm-run-all -p oxide-ci oxide-icons-ci -s tsc tinymce-rollup",
-    "ci-all": "npm-run-all -p eslint ci",
-    "local-ci": "npm-run-all ci-all -s phantomjs-test browser-test",
+    "ci": "npm-run-all -p oxide-ci oxide-icons-ci -p tsc \"tinymce-grunt dev\"",
+    "ci-all": "npm-run-all -p eslint ci -s tinymce-rollup",
+    "local-ci": "npm-run-all ci-all -s headless-test browser-test",
     "test": "run-s local-ci",
     "build": "npm-run-all -p oxide-icons-ci oxide-ci -s tinymce-grunt",
     "prepublishOnly": "run-s oxide-icons-ci",
     "browser-test": "yarn -s grunt browser-auto",
     "browser-test-manual": "yarn -s grunt browser-manual",
-    "phantomjs-test": "yarn -s grunt phantomjs-auto",
-    "phantomjs-test-manual": "yarn -s grunt phantomjs-manual",
+    "headless-test": "yarn -s grunt headless-auto",
```

Comment:
Would it be useful to have firefox headless here as well as a separate thing being able to run both in the console would be pretty nice. Maybe `headless-test-firefox` or something or is that out of scope of this PR since it's all about getting rid of phantomjs?

Reply:
That's a little harder, since the browser is hard coded in the phantom tests and I just renamed them. I think I can make it work, though

Reply:
Made it work, not as hard as it looked 👍

Reply:
Maybe this instead should do what we do for `browser-tests`? e.g there we add `--bedrock-browser` to control what browser it runs on.

Reply:
That's what I did - `--bedrock-browser` wasn't supported for headless. It is now (but requires a headless browser config).